src/Controller/UI/PersonFundingOrganizationController.php

```diff
@@ -1,55 +1,59 @@
 <?php
 
-namespace Pelagos\Bundle\AppBundle\Controller\UI;
+namespace App\Controller\UI;
 
-use Pelagos\Bundle\AppBundle\Form\PersonFundingOrganizationType;
+use App\Entity\PersonFundingOrganization;
+use App\Form\PersonFundingOrganizationType;
 
-use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use App\Handler\EntityHandler;
+use Symfony\Component\Routing\Annotation\Route;
 
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
 
 /**
  * The Research Group controller for the Pelagos UI App Bundle.
  */
-class PersonFundingOrganizationController extends UIController implements OptionalReadOnlyInterface
+class PersonFundingOrganizationController extends AbstractController
 {
     /**
      * The Person Funding Organization action.
      *
-     * @param string $id The id of the entity to retrieve.
+     * @param EntityHandler $entityHandler The entity handler.
+     * @param integer       $id            The id of the entity to retrieve.
      *
-     * @throws NotFoundException When the Funding Organization is not found.
+     * @throws NotFoundHttpException When the person funding organization is not found.
      *
-     * @Route("/person-funding-organization/{id}")
+     * @Route("/person-funding-organization/{id}", name="pelagos_app_ui_personfundingorganization_default")
      *
      * @return Response A Response instance.
      */
-    public function defaultAction($id = null)
+    public function defaultAction(EntityHandler $entityHandler, int $id = null)
     {
         // Checks authorization of users
         if (!$this->isGranted('ROLE_DATA_REPOSITORY_MANAGER')) {
-            return $this->render('PelagosAppBundle:template:AdminOnly.html.twig');
+            return $this->render('template/AdminOnly.html.twig');
         }
 
         $ui = array();
 
         if ($id !== null) {
-            $personFundingOrganization = $this->entityHandler->get('Pelagos:PersonFundingOrganization', $id);
+            $personFundingOrganization = $entityHandler->get(PersonFundingOrganization::class, $id);
 
-            if (!$personFundingOrganization instanceof \Pelagos\Entity\PersonFundingOrganization) {
-                throw $this->createNotFoundException('The Person Funding Organization was not found');
+            if (!$personFundingOrganization instanceof \App\Entity\PersonFundingOrganization) {
+                throw new NotFoundHttpException('The Person Funding Organization was not found');
             }
         } else {
-            $personFundingOrganization = new \Pelagos\Entity\PersonFundingOrganization;
+            $personFundingOrganization = new \App\Entity\PersonFundingOrganization;
```

Comment:
why the full path if we've included it in the use statements?

Reply:
This is how it was before, who known why. It should still work. It can be changed if desired.